docs/design/fabric-v2+/fpc-components.puml

```diff
@@ -12,14 +12,15 @@ title FPC Components
 top to bottom direction
 
 package protobufs {
+note "This proto message definition is deprecated\nand replaced with proto files in `protos/fpc"
 
 class CC_Parameters {
     + CC_Id string
     + CC_Version string
     + CC_Sequence int
     + Channel_Id string
     + MSPID string
-    + TLCC_MREnclave bytes
+    + TLCC_MREnclave string
```

Comment:
Two comments:
- from what we discussed earlier, i though we actually wanted to drop these classes from fpc-components.puml (and just reference place where it is defined: for this PR in `interfaces.md` but after PR #440 in `protocs/fpc/fpc.prot` to prevent duplications. Clearly the definition is currently inconsisent with the other places.  I guess that part got lost in my reviews, also probably confused as #440 and #439 touched fpc-components.puml. 
- regarding what is in CC_Parameters vs separately in Attestated_Data, there is a corresponding discussion in #420 or slack.  The strategy was if i can recall to separate what is common across peers from what is separate (hence MSPID now part of Host_Parameters). Arguably, channel hash and tlcc-mrenclave would still match the "common" part and could be part of CC_Parameters, not sure what the reasoning was but at least one interpretation of the protobufs as defined in interfaces.md/fpc.proto is that CC_Parameters are all the comman parameters as explicitly defined by the chaincode admins when creating the chaincode, i.e., essentially the parameters which are explicit or implicit in the enclave creation invoke (channel hash and tlcc-mrenclave are only then derived by asking tlcc ....

Reply:
I should not have touched `TLCC_MREnclave` here to trigger this discussion :) 
As Michael said, we deprecate the proto message description in `fpc-components.puml` and refer to the real protos. MSPID and TLCC_mrenclave are not part of cc_params anymore but still part of attested data, because MSPID is endorsing peer dependent, and tlcc_mrenclave contains "FPC infrastructure"-related data, both are not real chaincode parameters, still needed and kept inside the enclave.